admin_app/src/components/NavBar.tsx

```diff
@@ -52,8 +52,8 @@ const Logo = () => {
         component="img"
         src={logowhite.src}
         sx={{
-          height: 30,
-          width: 180,
+          height: 36,
+          aspect_ratio: 1200 / 214,
```

Comment:
@amiraliemami  I want to add this to the hotfix! I can add these commits but in case you can make them that'd also be great.